plugin/EstateList.php

```diff
@@ -409,6 +409,19 @@ public function estateIterator($modifier = EstateViewFieldModifierTypes::MODIFIE
 			$pEstateStatusLabel = $this->_pEnvironment->getEstateStatusLabel();
 			$recordModified['vermarktungsstatus'] = $pEstateStatusLabel->getLabel($recordRaw);
 		}
+		if ($this->_pWPOptionWrapper->getOption('onoffice-settings-title-and-description') == 0)
+		{
+			add_filter('document_title_parts', function() use ($recordModified) {
+				if (isset($recordModified["objekttitel"]))
+				{
+					return [$recordModified["objekttitel"]];
+				}
+				return [];
+			}, 10, 2);
+			add_action( 'wp_head', function () use($recordModified) {
+				echo '<meta name="description" content="' . esc_attr($recordModified["objektbeschreibung"] ?? null) . '" />';
```

Comment:
Isn't it way too late to trigger `wp_head` at this point?

Reply:
Still unsolved

Reply:
@jayay , i only know hook wp-head change tag meta description and i add priority with PHP_INT_MAX for this hook to it, and you review PR and suggest comment to finish this task .

Reply:
> Ok, since this seems to work (even though we don't understand why), we will use this `wp_head` hook and test it.

@jmaas-onoffice when display tag meta in <head>, we create 1 function hook in action wp_head, The wp_head action hook is triggered within the <head></head> section of the theme’s header.php template by the wp_head() function.

Reply:
We just thought that since we are in the template, we have already sent the `<head>`. So we thought that we couldn't add anything to the head at this point.

But if you say this works, then we'll just test it. 👍